### PR TITLE
[2.13.x] DDF-4010 Updating a documentation hyperlink to point to the most recent version of jts

### PR DESCRIPTION
**Pull request by Kjames5269**
#### What does this PR do?
Updates jts to the latest version

#### Who is reviewing it? 

@bakejeyner @brjeter 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@brendan-hofmann
@coyotesqrl

#### How should this be tested?
Download the jar. 👍 
#### What are the relevant tickets?
[DDF4010](https://codice.atlassian.net/browse/DDF-4010)
#### Checklist:
- [ 👍] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc [hunk 1]
```diff
@@ -124,7 +124,7 @@ Cloud installation instructions have been followed.
 
 The JARs can be found here:
 
-.. http://artifacts.codice.org/service/local/repositories/releases/content/org/codice/thirdparty/jts/1.12_1/jts-1.12_1.jar
+.. http://artifacts.codice.org/service/local/repositories/releases/content/org/codice/thirdparty/jts/1.14_1/jts-1.14_1.jar
```
[the remark this answers is not in the document]
Comment (Kjames5269) [reply distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc]: Nice catch 👍

**Comment by Kjames5269** [comment]
build now

**Comment by bakejeyner** [comment]
build now
(Jenkins issue)

**Comment by brjeter** [comment]
Heroed successfully ✅

**Comment by coyotesqrl** [comment]
Sidestepped a CI/static analysis _re-run_ as this is a documentation change that had already passed CI and heroing.

**Pull request merged**